crates/runtime/src/lib.rs

```diff
@@ -279,6 +282,32 @@ impl Runtime {
         Ok(())
     }
 
+    pub async fn load_models(&self) {
+        for model in self.app.read().await.models.clone() {
+            self.load_model(&model).await;
+        }
+    }
+
+    pub async fn load_model(&self, m: &spicepod::component::model::Model) {
```

Comment:
```suggestion
    pub async fn load_model(&self, m: &Model) {
```

Reply:
We actually have here `spicepod::component::model::Model ` and `runtime::model::Model` with similar names and conflicting. Updated to have it as `spicepod::component::model::Model as SpicepodModel;`